templates/vue/src/components/modals/NodeModal/forms/ContentForm/DragAndDropForm/index.vue

```diff
@@ -0,0 +1,203 @@
+<template>
+  <div>
+    <b-form-group>
+      <b>From buckets</b>
+      <b-form-checkbox v-model="useImages" data-qa="dragdrop-useImages">
+        Use Images
+      </b-form-checkbox>
+      <b-form-checkbox
+        v-if="useImages"
+        v-model="hideText"
+        data-qa="dragdrop-useImages"
+      >
+        Hide Text
+      </b-form-checkbox>
+      <drag-drop-bucket
+        v-for="(bucket, bucketIndex) in fromBuckets"
+        :key="bucket.id"
+        :node="node"
+        :question="question"
+        :index="bucketIndex"
+        :bucket="bucket"
+        :isFromBucket="true"
+        :fromBucketArray="fromBuckets"
+        :toBucketArray="toBuckets"
+        :useImages="useImages"
+        :data-qa="`from-bucket-${bucket.id}`"
+        @remove="fromBuckets.splice(bucketIndex, 1)"
+        @add="addNewFromBucketItem(bucketIndex)"
+      />
+      <b-button
+        class="addButton"
+        variant="primary"
+        squared
+        data-qa="add-from-bucket-button"
+        @click="addNewFromBucket"
+      >
+        Add a bucket
+      </b-button>
+    </b-form-group>
+    <b-form-group>
+      <b>To buckets</b>
+      <drag-drop-bucket
+        v-for="(bucket, index) in toBuckets"
+        :key="bucket.id"
+        :node="node"
+        :question="question"
+        :index="index"
+        :bucket="bucket"
+        :isFromBucket="false"
+        :fromBucketArray="fromBuckets"
+        :toBucketArray="toBuckets"
+        :useImages="useImages"
+        :data-qa="`to-bucket-${bucket.id}`"
+        @remove="toBuckets.splice(index, 1)"
+      />
+      <b-button
+        class="addButton"
+        variant="primary"
+        squared
+        data-qa="add-to-bucket-button"
+        @click="addNewToBucket"
+      >
+        Add a bucket
+      </b-button>
+    </b-form-group>
+  </div>
+</template>
+
+<script>
+import DragDropBucket from "./DragDropBucket"
+export default {
+  components: {
+    DragDropBucket,
+  },
+  props: {
+    node: {
+      type: Object,
+      required: true,
+    },
+    question: {
+      type: Object,
+      required: true,
+    },
+  },
+  data() {
+    return {
+      fromBuckets: [
+        {
+          id: 1,
```

Comment:
In most of the code, we have always been using `Helpers.createUUID()` function to generate unique IDs whenever we need them. It would be nice if we do the same thing here instead of starting with 1 and having to keep track of the latest id.